### DISCUSSION TITLE
How to access Log file from terminal

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5047] [Q&A] [answered]
From the web interface 

![image](https://github.com/stashapp/stash/assets/42092626/2d963c93-cb48-478b-8ee8-f82a06faa819)

### I am finding difficulty in accessing the log as it does not show the full log but only last few entries on the web interface.

On terminal I did, ` find . -name '*log*'` in the location where stash stores data, but there was no output.

**Comment by Maista6969** [comment, marked as answer]
I agree that the web interface is a poor way to view the logs: if you go into `Settings > System` and scroll to the bottom there's a section for Logging that will allow you to set up a log file 🙂

**Comment by MikeInCa** [comment]
I am newbie and could use some help on a related issue. I am running stashapp in a docker container. I can't figure out the naming convention of the folders. I need the correct path to get the log file activated. I would like the log to go in \docker\Stash\log folder but that doesn't work when I put it in the log path under Settings. Unfortunately I created the stash folder with a capital S "Stash" as well so maybe that is causing the issue?

**Reply by DogmaDragon** [reply]
The path needs to point to a file, not just the directory. Also make sure the path is a mounted volume, otherwise Stash won't be able to reach it.
